flow/project.py

```diff
@@ -3575,18 +4065,33 @@ def _main_exec(self, args):
             except KeyError:
                 operation = self._operations[args.operation]
 
-                def operation_function(job):
-                    cmd = operation(job).format(job=job)
+                def operation_function(*jobs):
+                    cmd = operation(jobs).format(jobs=jobs)
                     subprocess.run(cmd, shell=True, check=True)
 
         except KeyError:
             raise KeyError("Unknown operation '{}'.".format(args.operation))
 
-        if getattr(operation_function, '_flow_aggregate', False):
-            operation_function(jobs)
-        else:
-            for job in jobs:
-                operation_function(job)
+        try:
+            filter = operation_function._flow_select
+            grouper, sort = operation_function._flow_aggregate
+        except AttributeError:
+            filter = operation.cmd._flow_select
+            grouper, sort = operation.cmd._flow_aggregate
```

Comment:
This seems to be checking indirectly the operation class of the given operation. We should just check that directly then. Also, I think we want to allow for the group to determine the aggregation and selection behavior. This does not allow for that since we cannot override that in exec.